glTFLoader/Interface.cs

```diff
@@ -177,15 +234,33 @@ public static Byte[] LoadBinaryBuffer(this Gltf model, Func<string,Byte[]> exter
             return externalReferenceSolver(buffer.Uri);
         }
 
-        public static Stream OpenImageFile(this Gltf model, Func<string, Byte[]> externalReferenceSolver, int imageIndex)
+        /// <summary>
+        /// Opens a stream to the image referenced by a specific <code>Schema.Image</code>
+        /// </summary>
+        /// <param name="model">The <code>Schema.Gltf</code> model containing the <code>Schema.Image</code></param>
+        /// <param name="imageIndex">The index of the image</param>
+        /// <param name="externalReferenceSolver">An user provided lambda function to resolve external assets</param>
+        /// <returns>An open stream to the image</returns>
+        /// <remarks>
+        /// Images can be stored in three different ways:
+        /// - As stand alone files.
+        /// - As part of the binary chunk within a glb file.

```

Comment:
It would be more correct to say
> - As a part of binary buffer accessed via bufferView

E.g., it's perfectly valid to pack all images into one separate `.bin` file. Moreover, even GLB assets could have external references.